### DISCUSSION TITLE
Change Language

### DISCUSSION THREAD

**Discussion by phihungvohoang** [prasathmani/tinyfilemanager#1102] [Q&A]
How to custom language

**Comment by prasathmani** [comment]
you can add your own language to [https://github.com/prasathmani/tinyfilemanager/blob/master/translation.json](https://github.com/prasathmani/tinyfilemanager/blob/master/translation.json) file. to change the language, go to "settings" page and select the language.

<img width="1014" alt="image" src="https://github.com/prasathmani/tinyfilemanager/assets/11345598/8cff5fdd-6ec1-46da-8249-1d607b2234b2">
